src/scala/com/github/johnynek/bazel_deps/Target.scala

```diff
@@ -136,15 +137,31 @@ case class Target(
       if (!licenses.isEmpty) renderList(Doc.text("["), licenses.toList, Doc.text("]"))(quote)
       else Doc.empty
 
-    sortKeys(targetType, name.name, List(
-      visibilityDoc,
-      "deps" -> labelList(deps),
-      "licenses" -> renderLicenses(licenses),
-      "srcs" -> sources.render,
-      "jars" -> labelList(jars),
-      "exports" -> labelList(exports),
-      "runtime_deps" -> labelList(runtimeDeps),
-      "exported_plugins" -> renderExportedPlugins(processorClasses)
-    )) + renderPlugins(processorClasses, exports, generatesApi, licenses) + Doc.line
+    def generateTarget(neverlink: Boolean = false): Doc = {
+      var targetName = name.name
+      var targetArgs = List(
```

Comment:
can we avoid the `var` please? E.g.

```scala
val defaultArgs = List(...
val (targetName, targetArgs) =
  if (neverlink) (name.name + "_neverlink", defaultArgs :+ (("neverlink", Doc.text("1"))))
  else (name.name, defaultArgs)
```
we have tried to keep vars and mutation to an absolute minimum in this code.